src/expression/expression.ts

```diff
@@ -99,7 +99,7 @@ export interface AliasedExpression<T, A extends string>
   /**
    * Returns the alias.
    */
-  get alias(): A
+  get alias(): A | Expression<unknown>
```

Comment:
This change is backwards compatible since it only widens the type. All existing expressions are still valid.